backend/core/interfaces/zfs_repository.py

```
@@ -0,0 +1,160 @@
+    async def clone(self, name: SnapshotName, target_dataset: DatasetName) -> bool:
+        """Clone a snapshot to create a new dataset"""
+        pass
+    
+    @abstractmethod
+    async def send(self, name: SnapshotName, target_host: str, target_dataset: DatasetName) -> bool:
+        """Send snapshot to remote host"""
+        pass
+    
+    @abstractmethod
+    async def send_incremental(self, base_snapshot: SnapshotName, incremental_snapshot: SnapshotName, 
+                             target_host: str, target_dataset: DatasetName) -> bool:
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)